server/src/internalClusterTest/java/org/opensearch/repositories/RepositoriesServiceIT.java

```diff
@@ -173,16 +172,20 @@ public void testCreatSnapAndUpdateReposityCauseInfiniteLoop() throws Interrupted
         });
         thread.start();
 
-        logger.info("--> begin to reset repository");
-        repoSettings = Settings.builder().put("location", randomRepoPath()).put("max_snapshot_bytes_per_sec", "300mb");
-        OpenSearchIntegTestCase.putRepositoryWithNoSettingOverrides(
-            client().admin().cluster(),
-            repositoryName,
-            FsRepository.TYPE,
-            true,
-            repoSettings
-        );
-        logger.info("--> finish to reset repository");
+        try {
```

Comment:
The original intention of the unit test is to test that there will be no more dead loops. Occasional throwing exception is expected, as long as there is no more infinate loop.